review-database/postgres/data-generation/reviews.generator.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const getRandomIntInclusive = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
It looks like you're using this same function in two different files...not sure if it makes sense to do this, but in situations like this I would try to put this in a separate file and import it where you use it.